css-three-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  background-color: rgb(26 150 218);
+  color: white;
+  text-align: center;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  border-width: 40px;
+  padding: 40px 50px;
+  max-width: 1000px;
+  margin: auto;
+}
+
+.columns {
+  display: flex;
+}
+
+.column {
```

Comment:
`column` classes should be named after their size. This column is set to be a width of `33%`, so a good name would be `column-third`